Dockerfile.helm-unit-test

```diff
@@ -1,6 +1,6 @@
 # =================== CONTAINER FOR HELM UNIT TEST ===================
 
-FROM golang:alpine3.17 as conjur-k8s-helm-unit-test
+FROM golang:1.19-alpine as conjur-k8s-helm-unit-test
```

Comment:
Do we need this?

Reply:
following the 'boringcrypto' fix, we were required to upgrade things to 1.19